lib/shared/bucketing-assembly-script/__tests__/protobufVariableHelper.ts

```diff
@@ -0,0 +1,129 @@
+import path from 'path'
+import protobuf from 'protobufjs'
+import { SDKVariable, VariableType as VariableTypeStr } from '@devcycle/types'
+import {
+    variableForUser_PB, VariableType
+} from './bucketingImportHelper'
+
+// TODO replace all this PB importing + types once we import the pb file directly into static JS Classes
+const protoFile = '../protobuf/variableForUserParams.proto'
+const filePath = path.resolve(__dirname, protoFile)
+const root = protobuf.loadSync(filePath)
+
+const VariableForUserParams_PB = root.lookupType('VariableForUserParams_PB')
+if (!VariableForUserParams_PB) throw new Error('VariableForUserParams_PB not found')
+
+const SDKVariable_PB = root.lookupType('SDKVariable_PB')
+if (!SDKVariable_PB) throw new Error('SDKVariable_PB not found')
+
+type SDKVariable_PB_Type = {
+    _id: string
+    type: number
+    key: string
+    boolValue: boolean
+    doubleValue: number
+    stringValue: string
+}
+
+const pbSDKVariableToJS = (pbSDKVariable: SDKVariable_PB_Type): SDKVariable => {
+    if (pbSDKVariable.type === 0) {
```

Comment:
Are you avoiding using the explicit values instead of the VariableType enum here to just keep the tests isolated?

Reply:
yea basically I haven't figured out how to properly generate the protobuf Typescript classes yet from the `.proto` file, will do that as part of the NodeJS SDK PR, which will replace all of this.